Fix:
chm_http.c:167:32: error: implicit declaration of function 'inet_addr' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- src/chm_http.c.orig	2009-05-23 10:08:18.000000000 -0500
+++ src/chm_http.c	2021-03-13 13:11:43.000000000 -0600
@@ -40,6 +40,7 @@
 #endif
 
 /* includes for networking */
+#include <arpa/inet.h>
 #include <sys/socket.h>
 #include <sys/types.h>
 #include <netinet/in.h>
